assets/src/utils/common.ts

```diff
@@ -156,3 +156,16 @@ export const parseNumString = (item: string): string | number => {
 // Zips two arrays. E.g. zip([1,2,3], [4,5,6,7]) == [[1, 4], [2, 5], [3, 6]]
 export const zip = <T, U>(xs1: T[], xs2: U[]): [T, U][] =>
   xs1.reduce((acc, x, i) => (i > xs2.length - 1 ? acc : acc.concat([[x, xs2[i]]])), [] as [T, U][]);
+
+export const getFormattedVariables = (initStateFacts: Record<string, any>) => {
```

Comment:
this method should go in a different common type file, something related to adaptive lessons since *this* common file is generic for all things